tests/export/html/test_markup_compatibility.py

```diff
@@ -0,0 +1,91 @@
+# coding: utf-8
+
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+from pydocx.test import DocumentGeneratorTestCase
+from pydocx.test.utils import WordprocessingDocumentFactory
+from pydocx.openxml.packaging import MainDocumentPart
+
+
+class TableTestCase(DocumentGeneratorTestCase):
+    def test_textbox_with_content(self):
+        document_xml = '''
+            <p>
+                <r>
+                    <AlternateContent>
+                        <Fallback>
+                            <pict>
+                                <shape>
+                                    <textbox>
+                                        <txbxContent>
+                                            <p>
+                                                <r>
+                                                    <t>AAA</t>
+                                                </r>
+                                            </p>
+                                        </txbxContent>
+                                    </textbox>
+                                </shape>
+                            </pict>
+                        </Fallback>
+                    </AlternateContent>
+                </r>
+            </p>
+        '''
+
+        document = WordprocessingDocumentFactory()
+        document.add(MainDocumentPart, document_xml)
+
+        expected_html = '''
+            <p>
+                <p>
+                    AAA
+                </p>
+            </p>
+        '''
+        self.assert_document_generates_html(document, expected_html)
+
+    def test_textbox_with_content_outside_of_textbox(self):
```

Comment:
Continuation of https://github.com/CenterForOpenScience/pydocx/pull/203/files#r63035777 I have the tests written. However I don't have any idea what I need to do to break this up into multiple paragraphs. From what I can tell, the outside opening p tag has already happened. Which means I'll need to close it, do my thing, then re-open it. If this is the case, I have no idea how it can be done inline. I could probably build a post-processor, but it looks like we don't have those anymore.